R/prepare_weo_2024_scenario.R

```diff
@@ -6,35 +6,45 @@
 #'   `WEO2024_Extended_Data_World.csv` import.
 #' @param weo_2024_fig_chptr_3_raw A tidyxl data frame containing a raw import
 #'   of `WEO2024_Figures_Chapter_03.xlsx`.
-#' @param iea_global_ev_raw A data frame containing a raw `IEA Global EV Data
-#'   2024.csv` import.
+#' @param iea_global_ev_raw A data frame containing a raw import of 'electric-vehicle-sales-by-region-and-scenario-2030-and-2035.xlsx'
+#' sheet : electric-vehicle-sales-by-regio
 #' @param mpp_ats_raw A tidyxl data frame containing a raw import of `2022-08-12
 #'   - MPP ATS - RPK and GHG intensity.xlsx`.
-#'
+#' @param hybrid_methodology A character chain explaining how to handle Hybrid Vehicles
```

Comment:
a "character chain" is confusing, maybe a "character vector", "string value", "character object"?